tensorflow/contrib/seastar/seastar_server.cc

```diff
@@ -0,0 +1,112 @@
+seastar_client.h #include <cstdio>
+
+#include "boost/asio/ip/address_v4.hpp"
```

Comment:
Do we need the entire boost/asio library here? Better work out a port if you only need some utility functions.

Reply:
yes,

Reply:
Can you use inet_ntop instead? It is already referenced by TensorFlow code, so won't require additional dependency (i.e. it is already called here: https://github.com/tensorflow/tensorflow/blob/9590c4c32dd4346ea5c35673336f5912c6072bf2/tensorflow/core/platform/cloud/gcs_dns_cache.cc#L116)